### DISCUSSION TITLE
Spurious Heyu Signals?

### DISCUSSION THREAD

**Discussion by whitedavidp** [HeyuX10Automation/heyu#49] [Q&A]
![Screenshot_20221031-075306](https://user-images.githubusercontent.com/12023355/199047009-8b3897ad-36fc-4442-83bf-4f290d861638.png)

With regards the Heyu logtail shown in the attached screenshot...

Yesterday (10/30) @ 9:58am I got an ON command for house code A - no unit number.

At 10:03am I got an ON command for house code M, unit 1.

I live in a rural location and it is HIGHLY unlikely that there is another X10 installation anywhere nearby. I also have not seen spurious signals like this before.

Most of my stuff is on house code B, as you can see. I have nothing at all on M. But I DO have three things on house code A - all are unit 1:

1. A remote chime - this should receive only, right?
2. A plug-in universal module - this should receive only, right?
3. A plug-in interface module - this should send only, right?

Would the ON A command from 9:58am trigger the universal module set to A1? And would the same ON trigger the remote chime? I did not hear the chime (which I should have, I think I was in the area). And would it trigger the interface module?

What might be causing these spurious commands? I know that nothing in my setup should have triggered it. What can I do to prevent them?

Thanks

**Comment by whitedavidp** [comment]
BTW - I have Heyu 2.10.2 running in a chroot'ed Ubuntu 14.04 LTS on an Android TV box via Linux deploy. Not sure if this makes any difference. But...

Thanks again.

**Comment by jkrzyszt** [comment]
Please note that those unexpected signals appeared not alone, only close to or even interleaved with legal x10 transmission (attempts).
A new gadget plugged into a wall socket somewhere, or a dying x10 module, that magically distorts x10 signals?

**Reply by whitedavidp** [reply]
Thanks. It is hard to know what's up. I haven't seen this before. And there are always things being plugged and and removed. So I cannot isolate it at this point.
